lib/teiserver/coordinator/consul_commands.ex

```diff
@@ -239,17 +239,21 @@ defmodule Teiserver.Coordinator.ConsulCommands do
     state
   end
 
-  def handle_command(%{command: "splitlobby", senderid: senderid} = cmd, %{split: nil} = state) do
+  def handle_command(%{command: "splitlobby", remaining: rem, senderid: senderid} = cmd, %{split: nil} = state) do
     ConsulServer.say_command(cmd, state)
     sender_name = User.get_username(senderid)
 
+    min_players = get_default_min_players(rem)
```

Comment:
Given the contents and wider file I'd suggest inlining (don't make it a function, put it right here) this function.

Reply:
Done